src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -168,7 +168,8 @@ object AddDatasetMetadataToDeposit {
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
         case _                          =>
-          // TODO this case needs to be checked to not occur in the preconditions (see also line 32)
+          // TODO this case needs to be checked to not occur in the preconditions
+          // (see also comment in https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/master/src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala#L32)
```

Comment:
Fine with me as well, but you could also delete that commit by altering the history... And also, when you refer to a commit in a branch and you do a PR-squash-merge, I guess that commit doesn't exist anymore as well, right?

I'll add a commit SHA of the latest commit on master, expecting that this commit shouldn't vanish soon.